tools/purge_duplicate_rubyrep_triggers.rb

```diff
@@ -16,6 +16,9 @@
 GROUP BY relname
 SQL
 
+REGION = File.exists?("./REGION") ? File.read("./REGION").strip : "0"
```

Comment:
This would only be able to be run on the regional side of the replication because we rely on connecting to the database locally.